src/app/model/Rock.ts

```diff
@@ -0,0 +1,54 @@
+export class Rock implements PieceMovement {
```

Comment:
Brakuje Ci importów interfejsu np. Tez widze, ze zrobilem ten blad.

Reply:
to ciekawe, że nie rzucało żadnych błędów i śmigało niby normalnie :-)

(notka na jutro :-) )
dodam